### DISCUSSION TITLE
How to move the mini view from the bottom right to the top right

### DISCUSSION THREAD

**Discussion by lsig** [folke/noice.nvim#862] [Q&A]
Hi, I was wondering how do you config the mini fallback to appear in the top right corner instead of the bottom right. I have been reading the wiki and README, but I can't seem to figure it out. Any help is much appreciated.

I found this but https://github.com/folke/noice.nvim/discussions/835 but I could seem to get it to work. 

Thanks for an amazing plugin folke!

**Comment by ilias777** [comment]
All are described in the docs here &rarr; [docs](https://github.com/folke/noice.nvim/blob/8f1d9966762e62fa8788e5fb1a5f6a86784221d9/doc/noice.nvim.txt#L436) and in the views config file here &rarr; [views.lua](https://github.com/folke/noice.nvim/blob/8f1d9966762e62fa8788e5fb1a5f6a86784221d9/lua/noice/config/views.lua#L166)
In the `views.lua` file you can see the default mini settings and change the values in your noice config.

Put this in your noice configuration:

```lua
opts = {
    views = {
        mini = {
            position = {
                row = 1,
            },
        },
    },
},
```

<img width="1480" alt="mini-top-right" src="https://github.com/folke/noice.nvim/assets/39212564/3d560356-d4da-405c-9991-ac11ac8ec79c">
